### ISSUE TITLE
How does this function calculate grad:torch.nn.functional.cross_entropy 

### ISSUE THREAD

**Issue by laizewei** [pytorch/pytorch#15255]
How does this function calculate grad:torch.nn.functional.cross_entropy 
for example :out= input[i][target[i]], i=0to N-1,dout/dinput=?
Can you attach the derivative formula to the document?

cc @ezyang @albanD @zou3519 @gqchen @pearu @nikitaved @soulitzer @Lezcano @Varal7 @brianjo @mruberry

**Comment by zou3519** [comment]
I don't think we usually add derivative formulas to documentation

**Comment by albanD** [comment]
dout[i]/dinput[i][j] =1 if target[i] == j would be the gradient here.
Indeed we don't usually add the formulas to the doc.

We use github issues only for bugs or feature requests.
Please use the forum to ask questions: https://discuss.pytorch.org/
